### DISCUSSION TITLE
Stash or Export Items for DevilutionX or GOG version

### DISCUSSION THREAD

**Discussion by aspiringautisticcoder** [diasurgical/DevilutionX#3804] [Q&A] [answered]
I've been loving v1.3 of DevilutionX.
I found a Windforce with my MP Warrior. I want to give the Windforce to my MP Rogue.
Is there like an export items app or stash beta I can use to transfer the Windforce to my Rogue?
I actually tried Blue Scorpio's Diablo Saver with the GOG version and it wouldn't recognize my Diablo version even though it said in the notes that it works with GOG. And I tried it with DevilutionX just to see if it would work and it obviously didn't. Since the Diablo Saver app wasn't designed for it.

Thank you

**Comment by AJenbo** [comment, marked as answer]
DeviutionX saves are compatible with the original/GOG so you can simply transfer the save game. One way to transfer is to just start the game twice and set up a TCP game locally (join 127.0.0.1).

**Reply by aspiringautisticcoder** [reply]
Thanks. I found out that I could do this with DevilutionX as well in Windows 10 without the use of GOG. All I had to do was create a second desktop (virtual) in Windows 10. And I started the game on TCP on one desktop with my Warrior. And then started DevilutionX again with my Rogue on desktop 2. Joined 127.0.0.1. And handed off the Windforce to my Rogue.

**Reply by AJenbo** [reply]
You can also switch to a window by pressing alt+enter 🙂
